options/core/src/main/java/polanski/option/OptionAssertion.java

```diff
@@ -0,0 +1,63 @@
+package polanski.option;
+
+import android.support.annotation.NonNull;
+
+import polanski.option.function.Func1;
+
+public final class OptionAssertion<T> {
+
+    @NonNull
+    private final Option<T> actual;
+
+    OptionAssertion(@NonNull final Option<T> actual) {
+        this.actual = actual;
+    }
+
+    public void assertIsNone() {
+        if (!actual.isNone()) {
+            fail("Option was not None");
+        }
+    }
```

Comment:
I don't change this one, because I didn't think that anything else could be useful chained afterwards, but happy to make it return `OptionAssertion<T>` if you want.